### PR TITLE
PAYARA-3357 Fix Interactive Asadmin Prompt Errors - Payara4

### PR DESCRIPTION
**Pull request by Pandrex247**
Payara4 version of https://github.com/payara/Payara/pull/3528

### CODE REVIEW

**Comment by Pandrex247** [comment]
Jenkins test please

**Comment by Pandrex247** [comment]
Payara4 branch seems to alrgely be using Strings rather than chars - fixing now.

**Comment by Pandrex247** [comment]
Jenkins test please

File: nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java [hunk 1]
```diff
@@ -1164,8 +1164,8 @@ protected String getPassword(ParamModel opt, String defaultPassword,
             confirmationPrompt = strings.get("NewPasswordConfirmationPrompt", passwordName);
         }
 
-        char[] newpasswordAgain = readPassword(confirmationPrompt);
-        if (!Arrays.equals(newpassword, newpasswordAgain)) {
+        String newpasswordAgain = readPassword(confirmationPrompt);
+        if (newpassword.equals(newpasswordAgain)) {
```
Comment (svendiedrichsen) [inline nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java]: logic looks flipped. Missing a NOT (!) probably.

Comment (Pandrex247) [reply nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java]: Good spot sir!

**Comment by Pandrex247** [comment]
Jenkins test please

**Pull request merged**